app/controllers/application_controller.rb

```diff
@@ -62,14 +62,14 @@ def set_sentry_request_context
   def handle_pending_recrypt_request
     if pending_recrypt_request?
       pending_recrypt_request_message
-      redirect_to logout_login_path
+      redirect_to sessions_path, action: 'destroy'
```

Comment:
can't you just destroy the active session here? so the redirect would happen because the valid session is being reset.
seems a bit strange to me to redirect to a destroy action ...